ui/src/literature/components/ConferenceInfo.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Map, List } from 'immutable';
+import InlineList from '../../common/components/InlineList';
+import { getPageDisplay } from '../utils';
+
+class ConferenceInfo extends Component {
+  static renderAcronyms(acronyms) {
```

Comment:
can you make this non `static` access `acronyms` from props

Reply:
we have `static` render methods when we can't exact parameter from `props`. For example for functions that we use in `someCollection.map`